solr/README.md

```diff
@@ -42,3 +42,30 @@ additional settings, volumes, ports, etc.
 [Solr Documentation]: https://lucene.apache.org/solr/guide/7_1/
 [Solr Logging]: https://lucene.apache.org/solr/guide/7_1/configuring-logging.html
 [Solr]: https://lucene.apache.org/solr/
+
+## Changing versions
```

Comment:
Please add a space after the header for markdown standards.

Reply:
A trailing space character after "## Changing versions^"?

Reply:
Sorry, newline after the header.